dastardcommander/writing.py

```diff
@@ -65,8 +72,13 @@ def pathSelect(self):
             self.updatePath(result)
 
     def handleNumberWritten(self, d):
+        Nwritten = np.sum(d["NumberWritten"])
         self.label_numberWritten.setText("Number Written Total: {}\nNumber Written by Channel: {}".format(
-            np.sum(d["NumberWritten"]), d["NumberWritten"]))
+            Nwritten, d["NumberWritten"]))
```

Comment:
Hmm. I see what you mean. But I think I like it this way. There's no underlying "true value" for the target number. Every time the actual number written is updated, we test if it exceeds the target shown in the GUI (and stop if it does). So we're re-reading the GUI value every 2 seconds (or 1?).

Sure, it might be nice to know a percentage. Maybe I can add that...

Reply:
Okay. Done.